src/Hashids.net/Hashids.net.csproj

```diff
@@ -1,34 +1,34 @@
-        <PackageVersion>1.4.1</PackageVersion>
-        <Description>Generate YouTube-like hashes from one or many numbers. Use hashids when you do not want to expose your database ids to the user.</Description>
-        <PackageIconUrl>https://raw.githubusercontent.com/ullmark/hashids.net/master/hashids.png</PackageIconUrl>
-        <PackageProjectUrl>https://github.com/ullmark/hashids.net</PackageProjectUrl>
-        <PackageReleaseNotes>https://github.com/ullmark/hashids.net#changelog</PackageReleaseNotes>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <PackageTags>ids;hash</PackageTags>
-        <RepositoryType>git</RepositoryType>
-        <RepositoryUrl>https://github.com/ullmark/hashids.net</RepositoryUrl>
-        <TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>
-        <OutputType>Library</OutputType>
-        <LangVersion>latest</LangVersion>
-        <IncludeSymbols>true</IncludeSymbols>
-        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-        <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <!-- CS1591: Missing XML comment for publicly visible type or member -->
-        <NoWarn>$(NoWarn);CS1591</NoWarn>
-    </PropertyGroup>
+	<PropertyGroup>
+		<Title>Hashids.net</Title>
+		<Authors>Markus Ullmark</Authors>
+		<Version>1.4.1</Version>
+		<PackageVersion>1.4.1</PackageVersion>
+		<Description>Generate YouTube-like hashes from one or many numbers. Use hashids when you do not want to expose your database ids to the user.</Description>
+		<PackageIconUrl>https://raw.githubusercontent.com/ullmark/hashids.net/master/hashids.png</PackageIconUrl>
+		<PackageProjectUrl>https://github.com/ullmark/hashids.net</PackageProjectUrl>
+		<PackageReleaseNotes>https://github.com/ullmark/hashids.net#changelog</PackageReleaseNotes>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+		<PackageTags>ids;hash</PackageTags>
+		<RepositoryType>git</RepositoryType>
+		<RepositoryUrl>https://github.com/ullmark/hashids.net</RepositoryUrl>
+		<TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>
+		<OutputType>Library</OutputType>
+		<LangVersion>latest</LangVersion>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<!-- CS1591: Missing XML comment for publicly visible type or member -->
+		<NoWarn>$(NoWarn);CS1591</NoWarn>
+	</PropertyGroup>
 
-    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-      <PackageReference Include="System.Buffers" Version="4.5.1" />
-    </ItemGroup>
+	<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+		<PackageReference Include="System.Buffers" Version="4.5.1" />
+	</ItemGroup>
 
-    <ItemGroup>
-      <PackageReference Include="System.Memory" Version="4.5.4" />
-    </ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="System.Memory" Version="4.5.4" />
+	</ItemGroup>
```

Comment:
System.Memory package reference should also be conditional only for `netstandard2.0`